File: app/src/main/java/com/melean/taskstimetracker/record_tasks/view_holders/TaskViewHolder.java
package com.melean.taskstimetracker.record_tasks.view_holders;

import android.view.View;
import android.widget.TextView;

import com.melean.taskstimetracker.R;
import com.melean.taskstimetracker.record_tasks.adapters.BaseRecyclerAdapter;

public class TaskViewHolder extends BaseViewHolder{
    private TextView taskName;

    public TaskViewHolder(View view, BaseRecyclerAdapter.OnRecyclerItemClicked onRecyclerItemClicked) {
        super(view, onRecyclerItemClicked);
    }

    @Override
    public void onBindData(View view) {
        taskName = (TextView) view.findViewById(R.id.task_name);
    }

    public TextView getTaskName() {
        return taskName;
    }
}

File: app/src/main/java/com/melean/taskstimetracker/ui/view_holders/TaskViewHolder.kt
package com.melean.taskstimetracker.ui.view_holders

import android.view.View
import android.widget.TextView

import com.melean.taskstimetracker.R
import com.melean.taskstimetracker.ui.adapters.BaseRecyclerAdapter

class TaskViewHolder(view: View, onRecyclerItemClicked: BaseRecyclerAdapter.OnRecyclerItemClicked) : BaseViewHolder(view, onRecyclerItemClicked) {
    var taskName: TextView? = null
        private set

    override fun onBindData(view: View) {
        taskName = view.findViewById(R.id.task_name) as TextView
    }
}
